src/dioptra/restapi/v1/artifacts/schema.py

```diff
@@ -81,10 +128,40 @@ class ArtifactPageSchema(BasePageSchema):
     )
 
 
+class ShowFileTreeQueryParametersSchema(Schema):
+    """A schema for adding show_file_tree query parameters to an artifact
+    resource endpoint."""
+
+    showFileTree = fields.Boolean(
+        attribute="show_file_tree",
+        metadata=dict(description="Display files as a directory."),
+        load_default=None,
+    )
+
+
+class DownloadPathQueryParametersSchema(Schema):
```

Comment:
Consider renaming to something like `ArtifactContentsQueryParameterSchema` (especially if we add additional fields like `recurse`) since we are grouping a couple different parameters all related to the contents or files associated with the artifact.